src/components/organisms/WithNavigationBar/WithNavigationBar.scss

```diff
@@ -4,8 +4,69 @@
   padding-top: $navbar-height;
   padding-bottom: $footer-height;
 }
+
 .navbar-margin.fullscreen {
   height: 100%;
   width: 100%;
   overflow: auto;
 }
+
+.nav-logos {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-sparkle-logo,
+.nav-sparkle-logo_small {
+  position: relative;
+  margin-right: 12px;
+  &:hover {
+    width: 120px;
+  }
+  &.nav-sparkle-logo_small {
+    display: none;
+  }
```

Comment:
Since this is `display: none`, is there any reason for it to exist here?

Reply:
This style is not used I'll remove it.